WLActivityViewController/WLActivityViewController.m

```diff
@@ -183,18 +183,36 @@ - (void)showTitleView:(BOOL)animated {
     }
     _titleView.title = self.title;
     _titleView.translatesAutoresizingMaskIntoConstraints = NO;
-    [containerView addSubview:_titleView];
+    if ([[NSProcessInfo processInfo] isOperatingSystemAtLeastVersion:(NSOperatingSystemVersion){.majorVersion = 10}]) {
+        [contentView setContentInset:UIEdgeInsetsMake(50, 0, 0, 0)];
```

Comment:
Compilation error: No visible @interface for 'UIView' declares the selector 'setContentInset:'